### PR TITLE
Feature/abu 347

### PR DESCRIPTION
**Pull request by taylortom**
https://adaptlearning.atlassian.net/browse/ABU-347

### CODE REVIEW

**Comment by moloko** [comment]
+1

**Comment by oliverfoster** [comment]
+1

File: js/adapt-contrib-media.js [hunk 1]
```diff
@@ -54,7 +57,22 @@ define(function(require) {
             }
         },
 
-        // Used to check if the media should reset on revisit
+        // Overrides the default play/pause functionality to stop accidental playing on touch devices
+        setupPlayPauseToggle: function() {
+            this.mediaElement.player.options.clickToPlayPause = false;
```
Comment (dancgray) [inline js/adapt-contrib-media.js]: Should we check first to avoid a TypeError before setting this value?

**Comment by taylortom** [comment]
@dancgray would you mind testing https://github.com/taylortom/adapt-contrib-media/tree/feature/ABU-347-error-check?

I can't replicate the issue you describe, so a bit of a stab in the dark.

**Comment by dancgray** [comment]
@taylortom I'm still getting the error. It is occurring on iOS, as before. Do you want me to raise an issue once this (feature/ABU-347 branch) has been merged as it looks like it could be specific to my setup?

**Comment by taylortom** [comment]
Hmm ok. I'll have a test on iOS myself and see if I can figure it out, cheers.

**Comment by taylortom** [comment]
Seems the mediaElement.player is missing on iOS for some reason.

The branch I linked to above fixes this for me, although it means that the code won't execute on iOS, hence making this PR completely useless. 

I'll have a look into other solutions.

**Comment by dancgray** [comment]
+1 seems to fix the issue

**Comment by oliverfoster** [comment]
+1

**Comment by moloko** [comment]
+1

**Pull request merged**